microprofile-reactive-messaging-amqp-connector-example/scripts/generate-self-singned-certs.sh

```diff
@@ -0,0 +1,11 @@
+# Generate broker's key pair using obtained OpenShift cluster domain for `apps`
```

Comment:
there's a typo in the name, it should be `generate-self-signed-certs.sh`

Reply:
Fixed.